build/BuildPackages.cake

```diff
@@ -0,0 +1,35 @@
+public class BuildPackages
```

Comment:
Should add a `#load "./BuildPackage.cake"` here. It's not needed but it will make it explicit that this is needed.